docs: move cli commands above api in nav

jenniferfell opened the pull request:
## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ X] Docs have been added / updated (for bug fixes / features)


## PR Type
What kind of change does this PR introduce?

<!-- Please check the one that applies to this PR using "x". -->
```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[X ] Documentation content changes
[ ] angular.io application / infrastructure changes
[ ] Other... Please describe:
```

## What is the current behavior?
<!-- Please describe the current behavior that you are modifying, or link to a relevant issue. -->
Left navigation...

Release Information >
Quick Reference >
API
CLI Commands >

Issue Number: N/A


## What is the new behavior?
Left navigation...

Release Information >
Quick Reference >
CLI Commands >
API

Given that 100% of users need the API reference at some point, given the ease of access at the bottom of the nav, and given that it doesn't have any children...this feels better. 

## Does this PR introduce a breaking change?
```
[ ] Yes
[X ] No
```

<!-- If this PR contains a breaking change, please describe the impact and migration path for existing applications below. -->


## Other information

jenniferfell committed:
docs: move cli commands above api in nav

mary-poppins commented:
You can preview b72cd9d at https://pr26405-b72cd9d.ngbuilds.io/.

Splaktar commented on aio/content/navigation.json:
```diff
@@ -612,23 +612,22 @@
-      "url": "api"
-    },
     {
       "title": "CLI Commands",
```
Should this section just be called “CLI”? Since it includes the “Using the CLI” sub-section.

jenniferfell replied:
Changed "Using" to "Overview" to be more clear that this first entry isn't a guide to all the things you can do with the CLI; it's an introduction/overview to the CLI tool, commands, and syntax. "Using" is described throughout the docs, in the related guides such as Getting Started (using ng new), i18n, universal, routing, building, deployment, etc.

Splaktar reviewed (approved):
I like the CLI coming before the API (which I see as more advanced content).

mary-poppins commented:
You can preview b75273b at https://pr26405-b75273b.ngbuilds.io/.

mary-poppins commented:
You can preview c89570b at https://pr26405-c89570b.ngbuilds.io/.

jenniferfell committed:
fixup! docs: move cli commands above api in nav

mary-poppins commented:
You can preview 99d985e at https://pr26405-99d985e.ngbuilds.io/.

jenniferfell commented:
Changes: 
* Moved CLI above API in left nav (user vote was 6:1 in favor of this move)
* Renamed "Using the CLI" to "Overview" because this isn't a guide to all things you can do with the CLI. It's an overview/intro. 

@IgorMinar Ready for your review, with requests or approval. Thanks!

IgorMinar reviewed (approved):
ok. I admit. this is better :-)

IgorMinar commented:
caretaker note: ci is flaky
